docs/_posts/examples/9999-01-02-browsers.md

```diff
@@ -25,7 +25,7 @@ In your application, kick off a login flow with `ciscospark.authorize()`. This w
 
 Once the user logs in, we'll redirect them back to your app; the SDK will take care of grabbing the access token from the response and initializing with it.
 
-Your credentials will be cached to `localStorage` automatically. Previous to `ciscospark@0.7.0`, you were responsible for caching your credentials. For more details on the storage layer, take a look at [Storage Layer]({% post_url 9999-03-01-storage-layer %})
+Your credentials will be cached to `localStorage` automatically. Previous to `ciscospark@0.7.0`, you were responsible for caching your credentials. For more details on the storage layer, take a look at [Storage Layer]({{ site.baseurl }}{% post_url 9999-03-01-storage-layer %})
```

Comment:
interesting. kinda seems like they're taking the wrong opinion on that one, but if that's what's needed, LGTM.